pages/[slug].js

```diff
@@ -70,12 +70,14 @@ export default ({ event }) => (
         <Text variant="caption">
           {tt('{MM} {DD}, {YYYY}').render(new Date(event.start))}
         </Text>
-        { event.amaAvatar ? <Avatar size={128} sx={{ mt: 3  }} src={event.amaAvatar}></Avatar> : null }
+        {event.amaAvatar ? (
+          <Avatar size={128} sx={{ mt: 3 }} src={event.amaAvatar}></Avatar>
+        ) : null}
         <Text sx={{ my: [2, 3], fontSize: [2, 3] }}>{event.desc}</Text>
         <Button
           as="a"
           taget="_blank"
-          href={event.cal}
+          onClick={() => window.open(event.cal, '_blank')}
```

Comment:
instead of doing this, can you keep the `href=` and fix `taget` to be `target`? That should make it work

Reply:
fixed!